package/MDAnalysis/lib/util.py

```diff
@@ -30,11 +30,7 @@
 ---------------------
 
 .. autofunction:: filename
-.. function:: openany(directory[,mode='r'])
-
-   Context manager to open a compressed (bzip2, gzip) or plain file
-   (uses :func:`anyopen`).
-
+.. autofunction:: openany
```

Comment:
Does the context manager decorator preserve function docs nowadays (or is sphinx smarter)? I remember I wrote it explicitly because the decorator had mangled the docs.